packages/perspective-viewer-d3fc/src/js/axis/mainAxis.js

```diff
@@ -8,14 +8,14 @@
  */
 import * as d3 from "d3";
 import * as fc from "d3fc";
+import {linearExtent} from "../d3fc/extent/extent";

```

Comment:
Not sure what the intent was, but if you wanted to give the new component the same name as the d3fc one (which I guess you'll need to if you're replacing the d3fc one), then here you could do something like:

````javascript
import {extentLinear as customExtent} from "../d3fc/extent/extentLinear";
````